enrollment-server-onboarding/pom.xml

```diff
@@ -299,6 +299,43 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>embedded-db</id>
+            <dependencies>
+                <dependency>
+                    <groupId>org.liquibase</groupId>
+                    <artifactId>liquibase-core</artifactId>
+                </dependency>
+            </dependencies>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-resources-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>copy-resources</id>
+                                <phase>process-resources</phase>
+                                <goals>
+                                    <goal>copy-resources</goal>
+                                </goals>
+                                <configuration>
+                                    <outputDirectory>${basedir}/target/classes/db/changelog</outputDirectory>
+                                    <resources>
+                                        <resource>
+                                            <directory>${project.parent.basedir}/docs/onboarding/db/changelog</directory>
```

Comment:
👀 Pay attention to the structure of multi-war projects, `${project.parent.basedir}/docs/onboarding/db` versus `${project.parent.basedir}/docs/db`